namesilo.php

```diff
@@ -2929,21 +2963,37 @@ protected function getPrices()
         );
 
         if ($cache) {
-            return unserialize(base64_decode($cache));
+            $result = unserialize(base64_decode($cache));
         }
 
         Loader::loadModels($this, ['Currencies']);
 
-        $row = $this->getRow();
+        if (!$result) {
```

Comment:
This variable reference may cause an error depending on php configuration. Instead use:

```
        if (!isset($result)) {
```